libs/wingsdk/src/std/json.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Immutable Json
+ */
+export class Json {
+  /**
+   * Returns a specified element from the Json.
+   *
+   * @macro ($self$)[$args$]
+   *
+   * @param key The key of the element to return
+   * @returns The element associated with the specified key, or undefined if the key can't be found
+   */
+  public get(key: string): Json {
+    key;
+    throw new Error("Macro");
+  }
+}
+
+/**
+ * Mutable Json
+ */
+export class MutJson {
```

Comment:
What about accessing array elements? Maybe `getAt()` and `setAt()`?

Reply:
😅 index an element in an array! Who would need to do that? This is totally a use case I missed here, tunnel visioned on Maps. Ill add `getAt()` and `setAt()` apis to support this. Im sure this will evolve as `dot` and `[]` notations are added.